code/modules/surgery/organs/organ_external.dm

```diff
@@ -458,6 +465,18 @@ Note that amputating the affected organ does in fact remove the infection from t
 		tbrute = 3
 	return "[tbrute][tburn]"
 
+/obj/item/organ/external/proc/update_splints()
+	if(!(status & ORGAN_SPLINTED))
+		owner.splinted_limbs -= src
+		return
+	if(owner.step_count >= splinted_count + SPLINT_LIFE)
+		status &= ~ORGAN_SPLINTED //oh no, we actually need surgery now!
+		owner.visible_message("<span class=danger>[owner] screams in pain as their splint pops off their [src.name]!</span>","<span class=userdanger>You scream in pain as your splint pops off your [src.name]!")
```

Comment:
Your spans are broken... neither have the CSS type in quotes, and the second span doesn't get closed.

Also grammatically:
`as your splint pops off your [src.name]!`

should be

`as your splint pops off of your [src.name]!`

Reply:
`your [src.name]` should be `your [name]` as well

Reply:
:eyes: I'll take a look, I dunno why it worked when I tested it

Reply:
@Vivalas because `src` is implied, its unnecessary.

Reply:
The span stuff rather

Reply:
It'd probably not show the class - but either way for consistent styling with the rest of the repo, please do it. - but the not closing the span can/does lead to classes leaking into other things - which we definitely don't want.

Reply:
@SamHPurp Actually, this is perfectly valid HTML, IE will happily parse without any delimiters inside span tags. However, styling does dictate it requires the quotes.

Reply:
Yeah I wasn't certain, qualified it with 'probably' and then said the reason was more styling ;)

Nice to have confirmation though.